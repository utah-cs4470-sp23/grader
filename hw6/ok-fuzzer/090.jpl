show [((if (15.41 >= 28.41) then [93.02] else [39.84])[577] + (if true then (if (false || true) then (- 6.47) else 16.26) else ((- 39.77) % (- 38.96))))]

