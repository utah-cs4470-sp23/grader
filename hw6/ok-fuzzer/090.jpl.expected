(ShowCmd
 (ArrayLiteralExpr
  (ArrayType (FloatType) 1)
  (BinopExpr
   (FloatType)
   (ArrayIndexExpr
    (FloatType)
    (IfExpr
     (ArrayType (FloatType) 1)
     (BinopExpr
      (BoolType)
      (FloatExpr (FloatType) 15)
      >=
      (FloatExpr (FloatType) 28))
     (ArrayLiteralExpr (ArrayType (FloatType) 1) (FloatExpr (FloatType) 93))
     (ArrayLiteralExpr (ArrayType (FloatType) 1) (FloatExpr (FloatType) 39)))
    (IntExpr (IntType) 577))
   +
   (IfExpr
    (FloatType)
    (TrueExpr)
    (IfExpr
     (FloatType)
     (BinopExpr (BoolType) (FalseExpr) || (TrueExpr))
     (UnopExpr (FloatType) - (FloatExpr (FloatType) 6))
     (FloatExpr (FloatType) 16))
    (BinopExpr
     (FloatType)
     (UnopExpr (FloatType) - (FloatExpr (FloatType) 39))
     %
     (UnopExpr (FloatType) - (FloatExpr (FloatType) 38)))))))
